srcalc/src/main/java/gov/va/med/srcalc/service/ModelInspectionService.java

```diff
@@ -24,4 +24,16 @@
      * @return an ImmutableCollection, in arbitrary order
      */
     public ImmutableCollection<RiskModel> getAllRiskModels();
+
+    
+    /**
+     * Returns the {@link RiskModel} with the given ID.
+     * @return RiskModelS
+     */
+    public RiskModel getRiskModelForId(int modelId);    
+    
+    /**
+     * Saves the {@link RiskModel} 
+     */
+    public void saveRiskModel( final RiskModel model );    
```

Comment:
This method still doesn't belong in `ModelInspectionService`. See the interface doc comment: "All returned objects are intended for inspection only. This interface does not provide a means to persist any changes."